Makefile

```diff
@@ -221,6 +221,11 @@ clean_bindata:
 test: $(ADDON_ASSET_FILE)  ## Run unit tests
 	@go test -v -tags "$(BUILD_TAGS_SYSTEMTRAY)" -ldflags="$(VERSION_VARIABLES)" $(shell $(PACKAGES))
 
+.PHONY: coverage
+coverage: $(ADDON_ASSET_FILE)
+	rm -f out/coverage.txt
+	@go test -v -tags "$(BUILD_TAGS_SYSTEMTRAY)" -ldflags="$(VERSION_VARIABLES)" -coverprofile=out/coverage.txt -covermode=atomic $(shell $(PACKAGES))
```

Comment:
I got below result in `out/coverage.txt` when I executed `make coverage`. I am not sure how to interpret the result here. 
```
mode: atomic
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:33.47,34.21 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:37.3,38.17 2 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:34.21,36.4 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:38.17,40.4 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:44.13,47.2 2 1
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:49.31,51.16 2 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:54.2,55.12 2 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:58.2,59.16 2 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:62.2,62.20 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:65.2,67.45 3 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:51.16,53.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:55.12,57.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:59.16,61.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/unset.go:62.20,64.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:33.56,35.25 2 4
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:41.2,41.21 1 4
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:35.25,37.17 2 4
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:37.17,39.4 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:41.21,43.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:43.8,45.3 1 4
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:48.48,49.33 1 5
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:54.2,54.70 1 2
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:49.33,50.21 1 177
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:50.21,52.4 1 3
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:59.74,62.2 2 1
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:64.71,66.16 2 3
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:69.2,70.12 2 3
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:66.16,68.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:73.72,75.16 2 1
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:78.2,79.12 2 1
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:75.16,77.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:82.73,84.15 2 1
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:89.2,90.12 2 1
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:84.15,85.45 1 1
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:85.45,87.4 1 2
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:93.58,98.16 4 2
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:107.2,107.12 1 2
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:98.16,100.3 1 2
github.com/minishift/minishift/cmd/minishift/cmd/config/util.go:100.8,106.3 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/view.go:49.47,51.13 2 0
github.com/minishift/minishift/cmd/minishift/cmd/config/view.go:54.3,55.17 2 0
github.com/minishift/minishift/cmd/minishift/cmd/config/view.go:58.3,59.61 2 0
github.com/minishift/minishift/cmd/minishift/cmd/config/view.go:51.13,53.4 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/view.go:55.17,57.4 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/view.go:59.61,61.4 1 0
github.com/minishift/minishift/cmd/minishift/cmd/config/view.go:66.13,74.2 5 1
xxxxxxxxxxxxxxxxx
```